zt_frontend/src/components/FileExplorer.vue

```diff
@@ -79,6 +87,12 @@
     </template>
   </v-snackbar>
 
+  <FileFolderDownloadDialog
```

Comment:
All of the dialogs should live under the list item like this:

            <v-list-item
              v-if="<some_bool>"
              link
              prepend-icon="mdi-account-outline"
              title="Manage Access"
            >
              <ShareProjectDialog :project-name="item.name" />
            </v-list-item>
            
   Note the link directive